discovery/seata-discovery-raft/src/main/java/org/apache/seata/discovery/registry/raft/RaftRegistryServiceImpl.java

```diff
@@ -377,14 +377,23 @@ private static void acquireClusterMetaData(String clusterName, String group) thr
             try (CloseableHttpResponse httpResponse =
                 HttpClientUtil.doGet("http://" + tcAddress + "/metadata/v1/cluster", param, header, 1000)) {
                 if (httpResponse != null) {
-                    if (httpResponse.getStatusLine().getStatusCode() == HttpStatus.SC_OK) {
+                    int statusCode = httpResponse.getStatusLine().getStatusCode();
+                    if (statusCode == HttpStatus.SC_OK) {
                         response = EntityUtils.toString(httpResponse.getEntity(), StandardCharsets.UTF_8);
-                    } else if (httpResponse.getStatusLine().getStatusCode() == HttpStatus.SC_UNAUTHORIZED) {
-                        if (StringUtils.isNotBlank(USERNAME) && StringUtils.isNotBlank(PASSWORD)) {
-                            throw new RetryableException("Authentication failed!");
+                    } else if (statusCode == HttpStatus.SC_UNAUTHORIZED) {
+                        refreshToken(tcAddress);
+
+                        header.put(AUTHORIZATION_HEADER, jwtToken);
+                        httpResponse = HttpClientUtil.doGet("http://" + tcAddress + "/metadata/v1/cluster", param, header, 1000);
```

Comment:
I believe that simply refreshing the token is sufficient, and there is no need to retry the request, as the RetryableException will automatically retry after one second when it is thrown.

Reply:
yes i fixed